sklearn/covariance/_graph_lasso.py

```diff
@@ -355,8 +356,8 @@ class GraphicalLasso(EmpiricalCovariance):
     --------
     graphical_lasso, GraphicalLassoCV
     """
-
-    def __init__(self, alpha=.01, mode='cd', tol=1e-4, enet_tol=1e-4,
+    @_deprecate_positional_args
+    def __init__(self, alpha=.01, *, mode='cd', tol=1e-4, enet_tol=1e-4,
```

Comment:
I'm ok with this. I would also be ok with before the alpha. I have no arguments either way ;)

Reply:
I'm trying to minimize the amount of code these changes are going to break.